libs/remix-ui/workspace/src/lib/actions/workspace.ts

```diff
@@ -533,6 +533,33 @@ export const createNewBranch = async (branch: string) => {
   return promise
 }
 
+export const createSolidityGithubAction = async () => {
+  const actionYml = `
+  name: Running Solidity Unit Tests
+  on: [push]
+
+  jobs:
+    run_sol_contracts_job:
+      runs-on: ubuntu-latest
+      name: A job to run solidity unit tests on github actions CI
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - name: Environment Setup
```

Comment:
This step is not so necessary for a basic setup.

Reply:
ok I removed it